src/legal-officer/transaction-protection/PendingLocRequests.tsx

```diff
@@ -58,6 +60,14 @@ export default function PendingLocRequests(props: Props) {
         return null;
     }
 
+    function renderDetails(request: LocRequest): Child {
```

Comment:
What about factoring this function into an actual component which could be re-used (e.g. in RejectedLocRequests)?

Reply:
Done.